concent_api/core/tests/test_task_verification_result.py

```diff
@@ -201,7 +203,7 @@ def setUp(self):
         (self.PROVIDER_PRIVATE_KEY, self.PROVIDER_PUBLIC_KEY) = generate_ecc_key_pair()
         (self.REQUESTOR_PRIVATE_KEY, self.REQUESTOR_PUBLIC_KEY) = generate_ecc_key_pair()
 
-        task_to_compute = TaskToComputeFactory()
+        task_to_compute = TaskToComputeFactory(sig=ZERO_SIGNATURE)
```

Comment:
why do we need this?

Reply:
@dybi 
When we store to database single message then during serialization its gets own signature - if we serialialize it without any keys then to message is assign `fake_sign'. Let me show you how it's work. I'm gonna ignore parameters which we don't need here e.g: 
```python
store_subtask(
                task_to_compute=task_to_compute,
                report_computed_task=report_computed_task,
            )
```

We are here storing some `Subtask` with `task_to_compute` and `report_compute_task` which has as a nested message `task_to_compute`. When we serialize `task_to_compute` it will get `sig=fake_sig`. Same situation will be with `report_computed_task` beside `task_to_compute` which is nested in RCT. Nested message won't get its own `fake_sig` and that's why it will be different then `task_to_compute` stored separately. When `Subtask` model runs `clean()` method it will get `ValidationError` because `task_to_compute != report_computed_task.task_to_cumpute`. This is the only reason why we have to add `ZERO_SIGNATURE` here. It won't happen in normal cases because `TaskToCompute` is always signed.